src/query/api/v1/handler/prometheus/common.go

```diff
@@ -404,13 +362,33 @@ func RenderSeriesMatchResultsJSON(
 	jw.WriteString("success")
 
 	jw.BeginObjectField("data")
-	writeTags(jw, results)
+	jw.BeginArray()
+
+	for _, result := range results {
+		for _, tags := range result {
+			jw.BeginObject()
+			for _, tag := range tags.Tags.Tags {
```

Comment:
`tags.Tags.Tags` tags on tags on tags - can we possibly come up with a better naming convention?

Reply:
it's also a list of tags ;)

Agree we could do with a refactor but out of scope here